index.src.html

```diff
@@ -890,6 +893,24 @@ <h3 id="user-controls">User Controls</h3>
   active <a>referrer policy</a> on a page.
 </section>
 
+<section>
+  <h2 id="authoring">Authoring Considerations</h2>
+
+  <h3 id="unknown-policy-values">Unknown Policy Values</h3>
+
+  As described in <a section href="#determine-policy-for-token"></a>
+  and <a section href="#set-referrer-policy"></a>, unknown policy values
+  will be ignored, and when multiple sources specify a referrer policy,
+  the value of the latest one will be used. This makes it possible to
+  deploy new policy values. For example, suppose older user agents don't
+  understand the <code>unsafe-url</code> policy. A site can specify
```

Comment:
Nit: I'd suggest turning this into a "real" example by wrapping it in a `<div class="example">`, which will nicely offset it from the surrounding text.